### PR TITLE
Changelog for 0.2.0

### PR DESCRIPTION
**Pull request by nspope**
Summary of feature additions/bugfixes since v0.1.2

### CODE REVIEW

**Comment by nspope** [comment]
@petrelharp could you please take a look at the changelog when you get a chance? There's a pile of additions, not sure if I'm erring on the side of too much or too little detail.

**Review by mufernando — APPROVED** [review]
LGTM! Thanks @nspope for tackling this!!

**Review by jeromekelleher — APPROVED** [review]
Wow, great work @nspope, this is super helpful!

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,225 @@
...
+
+**Bug fixes**:
+
+- Parameters in the HomSap/Zigzag_1S14 model now match those in Schiffels &
+  Durbin (2014) (:user:`grahamgower`, :pr:`750`).
+
+- Recombination rate for DroMel chr4 changed to 0
+  (:user:`izabelcavassim`, :pr:`1092`).
+
+- Per-chromosome recombination rates for HomSap were incorrectly calculated
```
Comment (petrelharp) [inline CHANGELOG.rst:16]:
```suggestion
- Per-chromosome mean recombination rates for HomSap were incorrectly calculated
```

**Review by petrelharp — APPROVED** [review]
This is AMAZING, @nspope !

**Pull request merged**